tests/service/data/samename/tempet.fsproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
```

Comment:
Does this test anything valuable?  After all it is not using the locally built CompilerService.